suite-native/atoms/src/Image.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Image as RNImage, ImageProps as RNImageProps } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+type ImageProps = {
+    width?: number;
+    height?: number;
+} & RNImageProps;
+
+const imageStyle = prepareNativeStyle<{ width?: number; height?: number }>(
+    (_, { width, height }) => ({
+        width,
+        height,
+    }),
+);
+
+export const Image = ({ width, height, source, ...otherProps }: ImageProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <RNImage
+            {...otherProps}
+            source={source}
+            style={applyStyle(imageStyle, { width, height })}
```

Comment:
Yes, but right now it's unintended to use styling for Image component as there is no usage of Image that needs custom styles and I can't imagine why there would be for simple use cases. It can be added easily but I would just try to keep it simple. That's my reasoning for not adding styles there. 

But if you insist and have a good argument for it, I will add it - just didn't see reason to before.

Reply:
I guess if we don't merge styles, we could omit it from props though

Reply:
You don't need to merge styles just do `style={[applyStyle(imageStyle, { width, height }), style]}`

Reply:
[1d456aa](https://github.com/trezor/trezor-suite/pull/7889/commits/1d456aae554fa46269c007c6047352dfb9d28407)